src/gtk/window.cpp

```diff
@@ -3509,6 +3509,8 @@ void wxWindowGesturesData::Reinit(wxWindowGTK* win,
 
     wxASSERT_MSG( eventsMask == 0, "Unknown touch event mask bit specified" );
 
+    gtk_widget_add_events(widget, GDK_TOUCHPAD_GESTURE_MASK);
```

Comment:
wxGTK_HAS_GESTURES_SUPPORT is defined for GTK+ >= 3.14 but GDK_TOUCHPAD_GESTURE_MASK is available since 3.18, hence compilation fails for GTK+ older than 3.18.

Since